app/build.gradle

```diff
@@ -22,5 +22,10 @@ android {
         }
     }
 }
+dependencies {
+    implementation 'com.android.support.constraint:constraint-layout:+'
```

Comment:
@MaherSoua, can you delete this dependency. We already added in dependencies.gradle file.

Reply:
@rakeshpraneeth Ok